DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Centre/Administrator/AdministratorControllerTests.cs

```diff
@@ -91,16 +91,18 @@ public void Index_calls_expected_methods_and_returns_view()
         public void UnlockAccount_unlocks_account_and_returns_to_page()
         {
             // Given
-            A.CallTo(() => userDataService.GetAdminUserById(A<int>._)).Returns(UserTestHelper.GetDefaultAdminUser());
-            A.CallTo(() => userDataService.UpdateUserFailedLoginCount(A<int>._, A<int>._)).DoesNothing();
+            var adminUser = UserTestHelper.GetDefaultAdminUser();
+            A.CallTo(() => userDataService.GetAdminUserById(A<int>._)).Returns(adminUser);
+            A.CallTo(() => userService.ResetFailedLoginCountByUserId(A<int>._)).DoesNothing();
+            A.CallTo(() => userDataService.GetUserIdByAdminId(adminUser.Id)).Returns(1);

```

Comment:
* Replace `adminUser` with `var adminAccount = UserTestHelper.GetDefaultAdminAccount();`
* Remove `A.CallTo(() => userDataService.GetAdminUserById(A<int>._)).Returns(adminUser)` - it is not being called